### PR TITLE
QE-4 e2e | Registration 

### PR DESCRIPTION
**Pull request by qafromla**
https://qafromla.atlassian.net/browse/QE-4 
This PR includes : 
1. new test QALA-2 
2. new test QALA-6

### CODE REVIEW

File: cypress/e2e/registration.cy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+	it('should be able to sign up with valid credential, QALA-2', () => {
+		homePage.visit()
+		homePage.signUpButton.click()
+		RegisatrationPage.signUpNewAccount(username, email, password)
+		cy.contains(username).should('be.visible')
+	})
+
+	it('should not be able sign up with  an already registered email address and Username, QALA-6', () => {
+		homePage.visit()
```
Comment (andrewnovykov) [inline cypress/e2e/registration.cy.js]: use real credentials

File: cypress/pages/home.page.js [hunk 2]
```diff
@@ -2,6 +2,9 @@ class homePage {
 	get bannerTitle() {
 		return cy.get('.logo-font')
 	}
+	get signUpButton() {
+		return cy.get('[href="#/register"]')
+	}
 
 	visit() {
 		cy.visit('https://qafromla.herokuapp.com/')
```
Comment (andrewnovykov) [inline cypress/pages/home.page.js]: move to Base Url

**Review by andrewnovykov — CHANGES_REQUESTED** [review]

File: cypress/e2e/registration.cy.js (hunk 1 above)
Comment (qafromla) [reply cypress/e2e/registration.cy.js]: done

**Review by andrewnovykov — APPROVED** [review]
LGTM!

**Pull request merged**